src/Microsoft.Health.Fhir.TemplateManagement/Models/Digest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Models
+{
+    public class Digest
+    {
+        // Format of digest is: <algorithm>:<hex>
+        // e.g. sha256:d377125165eb6d770f344429a7a55379d4028774aebe267fe620cd1fcd2daab7
+        private static readonly Regex _digestRegex = new Regex("(?<algorithm>[A-Za-z][A-Za-z0-9]*([+.-_][A-Za-z][A-Za-z0-9]*)*):(?<hex>[0-9a-fA-F]{32,})");
+
+        public string Algorithm { get; set; }
+
+        public string Hex { get; set; }
+
+        public string Value { get { return string.Concat(Algorithm, ":", Hex); } }
+
+        public static List<Digest> GetDigest(string input)
```

Comment:
Are we getting multiple digests from a single string?

Reply:
We can return a list if input string contains multiple digests.